/users/PAS1585/llavez99/work/nbody/nBodyNN
Using TensorFlow backend.
WARNING:root:Limited tf.compat.v2.summary API due to missing TensorBoard installation.
2020-04-07 13:40:58.919924: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-04-07 13:40:58.931044: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2020-04-07 13:40:58.931573: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x561243cd3ae0 executing computations on platform Host. Devices:
2020-04-07 13:40:58.931597: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-04-07 13:40:59.027623: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
   file  eventID        m1  ...      dyf4      dyf5  Unnamed: 48
0   1.0  10001.0  4.828762  ... -0.739343  0.309672          NaN
1   1.0  10002.0  4.828762  ... -0.755528  0.325226          NaN
2   1.0  10003.0  4.828762  ... -0.771745  0.340821          NaN
3   1.0  10004.0  4.828762  ... -0.787995  0.356460          NaN
4   1.0  10005.0  4.828762  ... -0.804280  0.372141          NaN

[5 rows x 49 columns]
[[ 6.03865344e+01  8.27363079e+01  8.34884859e+01 ... -6.42913000e-01
   7.09420000e-01  1.35156250e+00]
 [ 9.33620150e+01  4.82251023e+01  3.45092725e+01 ...  7.92527000e-01
   3.14579000e-01  8.12109375e+00]
 [ 6.65195969e+01  3.38888246e+01  3.41536470e+01 ...  3.67206000e-01
  -6.55873000e-01  1.79296875e+00]
 ...
 [ 8.37399302e+01  5.06517880e+01  9.51991619e+01 ... -3.48375000e-01
  -4.80602000e-01  7.78515625e+00]
 [ 9.06237756e+01  8.48257820e+01  5.62686871e+01 ...  8.51349000e-01
   4.88150000e-02  8.99609375e+00]
 [ 5.19082213e+01  7.98107344e+01  4.77991354e+01 ...  3.12326000e-01
  -5.95955000e-01  6.32812500e-01]]
(8745, 26)
(8745, 26) (8745, 20)
Traceback (most recent call last):
  File "/users/PAS1585/llavez99/work/nbody/nBodyNN/nn_train_test5body.py", line 93, in <module>
    validation_data=(X_test,y_test))
  File "/users/PAS1585/llavez99/.local/lib/python3.6/site-packages/keras/engine/training.py", line 1154, in fit
    batch_size=batch_size)
  File "/users/PAS1585/llavez99/.local/lib/python3.6/site-packages/keras/engine/training.py", line 579, in _standardize_user_data
    exception_prefix='input')
  File "/users/PAS1585/llavez99/.local/lib/python3.6/site-packages/keras/engine/training_utils.py", line 145, in standardize_input_data
    str(data_shape))
ValueError: Error when checking input: expected dense_1_input to have shape (21,) but got array with shape (26,)

-----------------------
Resources requested:
nodes=1:ppn=10
mem=5000mb
-----------------------
Resources used:
cput=00:00:45
walltime=00:01:07
mem=0.345GB
vmem=2.710GB
-----------------------
Resource units charged (estimate):
0.018 RUs
-----------------------
